hosts_example.yml

```diff
@@ -235,7 +228,25 @@ kafka_broker:
   #   kafka_broker:
   #     datadir:
   #       - /var/lib/kafka/my-data
-
+  ## LDAP Connection Var
+  ## To configure LDAP for RBAC enablement, you will need to provide the appropiate properties to connect to your LDAP server
+  ## using the ldap_config varible.  The following is an example:
+  #ldap_config: |
```

Comment:
Nice, in our official docs lets remember to link the ldap docs, but this looks good!